src/MuTalk-Model/MutationTestingFixedNumberOfMutantsBudget.class.st

```diff
@@ -0,0 +1,18 @@
+Class {
+	#name : 'MutationTestingFixedNumberOfMutantsBudget',
+	#superclass : 'MutationTestingBudget',
+	#category : 'MuTalk-Model',
+	#package : 'MuTalk-Model'
+}
+
+{ #category : 'validation' }
+MutationTestingFixedNumberOfMutantsBudget >> check: partialCollection with: initialCollection [
+
+	^ partialCollection size >= constraint
+]
+
+{ #category : 'accessing' }
+MutationTestingFixedNumberOfMutantsBudget >> start [
```

Comment:
Maybe a comment of why this method is empty? :)

Also, why the `MutationTesting` prefix?
If we want to add a prefix to all classes I'd use a shorter one (`MT`?).
Otherwise, the choice so far was to not use a prefix...